SolarSystemBreakfast_colorize.rb

```diff
@@ -0,0 +1,78 @@
+# Projest Solar System. Breakfast level
+require "colorize"
+
+class Planet
+  attr_accessor :name, :distance, :radius, :mass, :rotation
+
+  def initialize(planets)
+    @name = planets[:name]
+    @distance = planets[:distance]
+    @radius = planets[:radius]
+    @mass = planets[:mass]
+    @rotation = planets[:rotation]
+  end
+
+  def to_s
+   "Planet: #{@name} distance: #{@distance} radius: #{@radius} mass: #{@mass} rotation: #{@rotaion}"
```

Comment:
You have a small typo with referencing `@rotation`, which results in that part of the string being blank.